R/helpers.R

```diff
@@ -364,14 +364,16 @@ create_dummy_package_full <- function(mn, title = "A Dummy Package") {
                            path = "dummy1.R",
                            format_id = "application/R")
 
+  unlink(c("dummy1.csv", "dummy2.csv", "dummy1.jpg", "dummy1.R"))
```

Comment:
This is fine as is, really, but I think it's easier to use `tempfile` which means you can create at will and don't have to deal with deleting files since the OS will delete the files eventually once the R process exits.

Reply:
yeah I agree - this kind of thing is all over the code (especially the tests). I didn't write it and don't feel much incentive to go back and fix it at this point though